code/GridFieldDuplicateRow.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Duplicate the selected row
+ *
+ * @author Gabriele Brosulo <gabriele@brosulo.net>
+ * @creation-date 21-apr-2017
+ */
+class GridFieldDuplicateRow implements GridField_ColumnProvider, GridField_ActionProvider {
```

Comment:
Yes, it's the same. I didn't find it on addons.silverstripe.org because I was searching for "clone" or "duplicate".